src/db/dbrc.js

```diff
@@ -0,0 +1,50 @@
+import debug from 'debug';
+import db from './db';
+
+const logger = debug('iReporter::initdb:');
+const Models = {
+  recordSchema: `
+    CREATE TABLE IF NOT EXISTS users
+      (
+       id SERIAL PRIMARY KEY,
+       firstname VARCHAR NOT NULL,
+       lastname  VARCHAR NOT NULL,
+       othernames VARCHAR,
+       username  VARCHAR UNIQUE NOT NULL,
+       email  VARCHAR NOT NULL,
+       phone  VARCHAR,
+       registered TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+       is_admin BOOLEAN DEFAULT 'false'
+      )`,
+
+  userSchema: `
+    CREATE TABLE IF NOT EXISTS records
```

Comment:
is this meant to be `userSchema` or `recordSchema`?

Reply:
should be `recordShema`   fixed in #32